Sources/Testing/Events/Recorder/Event.Symbol.swift

```diff
@@ -0,0 +1,171 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+extension Event {
+  /// An enumeration describing the symbols used as prefixes when recording an
+  /// event.
+  public enum Symbol: Sendable {
```

Comment:
The name "Symbol", on its own in this nested context, feels a little unclear. Can we consider alternatives? I suspect people won't need to type the name often, and will typically just receive an instance of this value, so maybe a longer name like `Event.RecorderMessageSymbol` would be acceptable?

Reply:
It's the symbol to use for an event, no?

I think putting it in the new `@_spi` category plus being nested in `Event` is a pretty good hint.